src/Payments.sol

```diff
@@ -296,24 +418,47 @@ contract Payments is
         Account storage payer = accounts[rail.token][rail.from];
         Account storage payee = accounts[rail.token][rail.to];
 
+        uint256 oldRate = rail.paymentRate;
+
+        // For terminated rails, can only reduce rate, not increase it
+        if (rail.terminationEpoch > 0) {
+            require(
+                newRate <= oldRate,
+                "failed because terminated at epoch: cannot increase rate on terminated rail"
+            );
+            require(
+                oneTimePayment <= rail.lockupFixed,
+                "failed because terminated at epoch: one-time payment exceeds fixed lockup"
+            );
```

Comment:
This is already required in all cases, right?

Reply:
Done.

Reply:
Also simplified the asserts here.